openprocurement/auction/tests/test_utils.py

```diff
@@ -59,3 +62,76 @@ def test_get_bidder_id_not_cache_200(self, app):
             assert get_bidder_id(app, sess) == TEST_BIDDER_DATA
             assert app.logins_cache[REMOTE_OAUTH] == TEST_BIDDER_DATA
             app.remote_oauth.get.assert_called_once_with('me')
+
+
+class TestSortingByAmount(object):
+    # do we need this test ?
+    def test_one_bid(self):
+        test_bids = [{'amount': 3955.0, 'bidder_id': 'df1', 'time': '2015-04-24T11:07:30.723296+03:00'}]
+
+        expected_bids = [{'amount': 3955.0, 'bidder_id': 'df1', 'time': '2015-04-24T11:07:30.723296+03:00'}]
+
+        assert sorting_by_amount(test_bids) == expected_bids
+        assert sorting_by_amount(test_bids, reverse=False) == expected_bids
+
+    def test_bids_no_features(self):
+        test_bids = [
+            {'amount': 3955.0, 'bidder_id': 'df1', 'time': '2015-04-24T11:07:30.723296+03:00'},
+            {'amount': 3966.0, 'bidder_id': 'df2', 'time': '2015-04-24T11:07:30.723296+03:00'},
+            {'amount': 3955.0, 'bidder_id': 'df4', 'time': '2015-04-23T15:48:41.971644+03:00'},
+        ]
+
+        expected_bids = [{'amount': 3966.0, 'bidder_id': 'df2', 'time': '2015-04-24T11:07:30.723296+03:00'},
+                         {'amount': 3955.0, 'bidder_id': 'df1', 'time': '2015-04-24T11:07:30.723296+03:00'},
+                         {'amount': 3955.0, 'bidder_id': 'df4', 'time': '2015-04-23T15:48:41.971644+03:00'}]
+
+        assert sorting_by_amount(test_bids) == expected_bids
+        assert sorting_by_amount(test_bids, reverse=False) == expected_bids[::-1]
+
+    def test_bids_only_features(self):
+        test_bids = [
+            {'amount_features': 0.04, 'bidder_id': 'df1', 'time': '2015-04-24T11:07:30.723296+03:00'},
+            {'amount_features': 0.06, 'bidder_id': 'df2', 'time': '2015-04-24T11:07:30.723296+03:00'},
+            {'amount_features': 0.05, 'bidder_id': 'df4', 'time': '2015-04-23T15:48:41.971644+03:00', },
+        ]
+
+        expected_bids = [{'bidder_id': 'df2', 'amount_features': 0.06, 'time': '2015-04-24T11:07:30.723296+03:00'},
+                         {'bidder_id': 'df4', 'amount_features': 0.05, 'time': '2015-04-23T15:48:41.971644+03:00'},
+                         {'bidder_id': 'df1', 'amount_features': 0.04, 'time': '2015-04-24T11:07:30.723296+03:00'}]
+
+        assert sorting_by_amount(test_bids) == expected_bids
+        assert sorting_by_amount(test_bids, reverse=False) == expected_bids[::-1]
+
+    # do we need this test ?
```

Comment:
There is no possibility to have two bids with and without 'amount_features'. So, remove this test.